src/examples/src/widgets/popup-confirmation/styles/Basic.m.css

```diff
@@ -0,0 +1,15 @@
+.root {
```

Comment:
can you put all these styles in the `root` class please? I think you just need to flex / center and set width / height to 100%